### PR TITLE
Gabrielle Meyers hw01

### PR DESCRIPTION
**Pull request by gmeyers405**
I added a bio to the READ.ME file by editing the R script in RStudio (top left box), saving these changes, committing the changes, and pushing the changes to Github. I found the video posted on the homework page on the class site to be extremely helpful in figuring out how to do this. I introduced myself and gave an idea of my life at home and at school. To demonstrate Markdown competency, I used the following elements:
- headers
- emphasis (bold and italic)
- a list
- an embedded link (I thought it would be humorous to include my LinkedIn, as well as demonstrate my ability to embed links)

Overall, this assignment was not too challenging. The video helped with the overall process, and the Github guide to Markdown was very clear. At first, I had a little trouble because I hadn't realized how important spaces were (coding for #hw01 instead of # hw01), so I'm glad I figured that out before turning this assignment in. For the Github workflow, I wasn't sure if I was supposed to make one commit for the entire READ.ME document or make a commit for each change, so I made about one commit per section. Overall, I feel like I'm getting the hang of both the Github workflow and Markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is Gabrielle Meyers, but most people call me Gabby.
+
+## Quick Facts
+* I'm a rising third year at the College.
+* I'm majoring in Public Policy and History.
+* I'm from Long Island, NY, where I currently live with my family and my dog, Ella.
```
Comment (bensoltoff) [inline README.md:8]: Nice use of itemized list! I recommend leaving a blank line between different Markdown elements, like this:

```
## Quick Facts

* I'm a rising third year at the College.
* I'm majoring in Public Policy and History.
* I'm from Long Island, NY, where I currently live with my family and my dog, Ella.
```

It avoids rendering issues in more complex documents

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+## My Life At School
+I lived in Jannotta House in South. I am a copyeditor for the *Maroon* and I also write for the *Gate*. I was an AC for one of the ChoMUN committees before our competition was cancelled due to the coronavirus. Furthermore, I volunteer through Alpha Phi Omega (the community service fraternity) and Supplies for Dreams.
+
+## My Life at Home
+I ended up enrolling for this class after being accepted into the **Summer Institute for Social Science Research Methods (SISRM).** In addition to taking this class, I also attend daily workshops while working with a faculty member on their research project. I am taking this class to get better at using R, since I'm currently primarily a Stata user. This summer, among other things, I hope to get better at using several data analysis software systems. In my spare time, I like to read.
```
Comment (bensoltoff) [inline README.md:14]: This course will definitely help on the programming side

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |         x  |              |            |
| **Coding strategy**         |        x   |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |        x   |              |            |
| **Reproducibility**         |    x       |              |            |

* Good mix of Markdown elements

**Comment by wgraif** [comment]
You clearly proved that you understand how to do exactly what you said you did. The headers, italics, bold, and bullets (maybe a sub-bullet addition would have been cool by using an indented plus, per https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf , but you got the idea) all look great. The link also works great (I added you on LinkedIn!). The image is certainly the most challenging of the five elements offered (the one that I personally struggled with) as you have to make sure that you reference both the directory in R, and have that same directory uploaded on GitHub for it to display here. Your descriptions are also all very good. I don't think that you have to commit as frequently as you did. Frequent commits are useful if you are working on different computers and want to store your progress on the cloud, or if you are working with someone else (for the same reason). But there isn't really a downside other than the time spent doing it. This small amount of text a few extra times is not going to overwhelm GitHub's (massive) storage space or anything.

**Comment by sandramauro** [comment]
I liked your use of headers and bold font to break up the different sections in the assignment. You also demonstrated how to use the list well and how to use the link well. Your text was also well organized and easy to read. I really liked your short biography!

I saw that you did't include an image, so I am including my code for how to add one in case it's useful:

**Pull request closed without merging**